pkgs/by-name/ne/nexusmods-app/package.nix

```
@@ -44,12 +39,13 @@ buildDotnetModule rec {
 
   preConfigure = ''
     substituteInPlace Directory.Build.props \
-      --replace '</PropertyGroup>' '<ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles></PropertyGroup>'
+      --replace-fail '</PropertyGroup>' '<ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles></PropertyGroup>'
   '';
 
-  postPatch = ''
-    ln --force --symbolic "${lib.getExe _7zzWithOptionalUnfreeRarSupport}" src/ArchiveManagement/NexusMods.FileExtractor/runtimes/linux-x64/native/7zz
-  '';
+  dotnetBuildFlags = [
+    "--property:INSTALLATION_METHOD_FLATPAK=1"
+    "--property:NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR=1"
+  ];
```

You're welcome to ignore this suggestion if you think it's overkill

```suggestion
  dotnetBuildFlags = [
    # Tell the app it is a distro package, e.g. affects wording in update prompts.
    # TODO: renamed to INSTALLATION_METHOD_PACKAGE_MANAGER after 0.5.1
    "--property:INSTALLATION_METHOD_FLATPAK=1"

    # Don't include upstream's 7zz binary; we want the nixpkgs version.
    "--property:NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR=1"
  ];
```